### DISCUSSION TITLE
search for multiple keywords to highlight

### DISCUSSION THREAD

**Discussion by zundaren** [sourcelair/xterm.js#4454] [Q&A] [answered]
Currently term.select() can only highlight one keyword, hope to add search keywords, such as ABA, select("A") -> two A highlighting

**Comment by Tyriar** [comment, marked as answer]
The search addon does this already. You can check out the source of the addon for how to use decorations to do special highlighting like this: https://github.com/xtermjs/xterm.js/blob/master/addons/xterm-addon-search/src/SearchAddon.ts

**Reply by zundaren** [reply]
thanks
